patches/server/0477-Add-methods-to-get-translation-keys.patch

```diff
@@ -174,5 +174,16 @@ index fab3063fffa959ac7f0eb5937f2fae94d11b6591..9292a0119499d14c9ed170999ac3b8df
 +            org.bukkit.block.Biome bukkit = org.bukkit.block.Biome.valueOf(nms.getKey().location().getPath().toUpperCase());
 +            Assert.assertEquals("translation key mismatch for " + bukkit, nms.getKey().location().toLanguageKey("biome"), bukkit.translationKey());
 +        }
++    }
++
++    @Test
++    public void testInventoryType() {
++        for (org.bukkit.event.inventory.InventoryType type : org.bukkit.event.inventory.InventoryType.values()) {
++            if (type == org.bukkit.event.inventory.InventoryType.MERCHANT || type == org.bukkit.event.inventory.InventoryType.COMPOSTER) {
++                continue;
++            }
++            Assert.assertNotEquals(type + " has an unresolved translation key",type.translationKey(), net.kyori.adventure.text.serializer.plain.PlainTextComponentSerializer.plainText().serialize(net.kyori.adventure.text.Component.translatable(type)));
```

Comment:
Missing space between the first and second argument. I think another test here that would compare and check that the hardcoded values are equals to their en_us translation might be convenient (this is especially useful if the defaultTitle is not updated see above)

Reply:
Not a good way of dynamically getting the en_us values, so it'd just be more hardcoding I think.